### PR TITLE
Style Bash PS1

### PR DESCRIPTION
**Pull request by Cyb10101**

### CODE REVIEW

File: .shell-methods.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Style bash prompt
+function stylePS1 {
+    VAR_HOSTNAME=`hostname`
+    if [ ! -z "$1" ]; then
+        VAR_HOSTNAME="${1}"
+    fi;
+
+    PS1='$(userTerminalTitlePwd)\[\e[0;36m\][$(userColorUser)\u\[\e[0;36m\]@\[\e[1;34m\]${VAR_HOSTNAME}\[\e[0;36m\]: \[\e[0m\]\w\[\e[0;36m\]]\[\e[0;36m\]> $(userColorUser)\n\$\[\e[0m\] ';
+    if [ -f $(which git) ]; then
+        PS1='$(userTerminalTitlePwd)\[\e[0;36m\][$(userColorUser)\u\[\e[0;36m\]@\[\e[1;34m\]${VAR_HOSTNAME}\[\e[0;36m\]: \[\e[0m\]\w\[\e[0;36m\]]\[\e[0;33m\]$(__git_ps1)\[\e[0;36m\]> $(userColorUser)\n\$\[\e[0m\] ';
```
Comment (Kanti) [inline .shell-methods.sh]: can we remove the duplicate characters from these lines?

Comment (Cyb10101) [reply .shell-methods.sh]: see .shell-methods function renderGitBranch

File: .shell-methods.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+# Style bash prompt
+function stylePS1 {
+    VAR_HOSTNAME=`hostname`
+    if [ ! -z "$1" ]; then
+        VAR_HOSTNAME="${1}"
+    fi;
+
+    PS1='$(userTerminalTitlePwd)\[\e[0;36m\][$(userColorUser)\u\[\e[0;36m\]@\[\e[1;34m\]${VAR_HOSTNAME}\[\e[0;36m\]: \[\e[0m\]\w\[\e[0;36m\]]$(renderGitBranch)\[\e[0;36m\]> $(userColorUser)\n\$\[\e[0m\] ';
```
Comment (Kanti) [inline .shell-methods.sh]: can we change this line up, so it is more readable?

Comment (Cyb10101) [reply .shell-methods.sh]: Color variables added

File: .shell-methods.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+# Style bash prompt
+function stylePS1 {
+    VAR_HOSTNAME=`hostname`
+    if [ ! -z "$1" ]; then
+        VAR_HOSTNAME="${1}"
+    fi;
+
+    PS1='$(userTerminalTitlePwd)${COLOR_CYAN}[$(userColorUser)\u${COLOR_CYAN}@${COLOR_LBLUE}${VAR_HOSTNAME}${COLOR_CYAN}: ${COLOR_RESET}\w${COLOR_CYAN}]$(renderGitBranch)${COLOR_CYAN}> $(userColorUser)\n\$${COLOR_RESET} ';
```
Comment (Kanti) [inline .shell-methods.sh]: it would be very nice if this line could be shorter.
Can you (@Cyb10101) split this into multiple lines?

**Comment by ChrisB9** [comment]
Closing this, as there hasn't been any update since month and currently it adds a lot of overhead to the setup just for some pre-configured colors

**Pull request closed without merging**